node/service/src/chain_spec/karura.rs

```diff
@@ -46,7 +46,7 @@ fn karura_properties() -> Properties {
 	let mut properties = Map::new();
 	let mut token_symbol: Vec<String> = vec![];
 	let mut token_decimals: Vec<u32> = vec![];
-	[KAR, KUSD, KSM, LKSM].iter().for_each(|token| {
+	[KAR, KUSD, KSM, LKSM, BNC].iter().for_each(|token| {
```

Comment:
not sure if BNC fits here, @xlc could you confirm?

Reply:
I think we can have it here. Helps client to know decimals of BNC.
But also need to update https://github.com/AcalaNetwork/Acala/blob/master/resources/karura-dist.json